src/argilla/server/schemas/v1/questions.py

```diff
@@ -86,3 +90,21 @@ class Question(BaseModel):
 
     class Config:
         orm_mode = True
+
+
+class TextQuestionSettingsUpdate(UpdateSchema):
+    use_markdown: Optional[bool]
```

Comment:
Regarding this schema definition, since we have only one single attribute, maybe we should force provide it when settings are sent as part of the updated body:
this
````json
{ 
   "settings": {}
}
````
is a syntactically valid update body, but maybe makes no much sense, since is equivalent to 
````json
{}
````

Maybe, we should force API consumers to provide a value for `use_markdown` if settings are part of the request body:
```json
{
   "settings": {
      "use_markdown": ...
   } 
}
```
In that case, the schema definition could be:

```python
class TextQuestionSettingsUpdate(UpdateSchema):
    use_markdown: bool
    __non_explicit_none__ = {"use_markdown"} # I think this shouldn't be needed 
```

We should review the FieldsSettings update to align it with this approach.